doc-files/release-notes-15.0.1.md

```diff
@@ -0,0 +1,21 @@
+# Release Notes for JavaFX 15.0.1
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 15.0.1 release. As such, they complement the [JavaFX 15 Release Notes](https://github.com/openjdk/jfx/blob/jfx15/doc-files/release-notes-15.md).
+
+JavaFX 15.0.1 requires JDK 11 or later.
+
+## List of Fixed Bugs
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8255064](https://bugs.openjdk.java.net/browse/JDK-8255064) | Cherry pick GTK WebKit 2.28.4 changes | web          
+[JDK-8255063](https://bugs.openjdk.java.net/browse/JDK-8255063) | Cherry pick GTK WebKit 2.28.3 changes | web          
+[JDK-8255062](https://bugs.openjdk.java.net/browse/JDK-8255062) | Update to 610.1 version of WebKit     | web      
```

Comment:
These are the JBS IDs of the backport records. We normally use the main bug ID everywhere. The rest looks fine.

Reply:
That is fixed now.